### PR TITLE
BUGFIX: SD-743 get length of measures in secondary yAxis config when it existed

### PR DESCRIPTION
**Pull request by lvkhanh**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2665
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2465

# Related Jira tasks

### CODE REVIEW

**Comment by lvkhanh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1292.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5450/ : SUCCESS in 8m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/949/ : SUCCESS in 17s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/950/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2248/ : SUCCESS in 32m 32s

**Comment by lvkhanh** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2821/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/951/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2249/ : SUCCESS in 15m 36s

**Comment by ptquang86** [comment]
LGTM

File: src/helpers/MdObjectHelper.ts [hunk 1]
```diff
@@ -113,7 +113,8 @@ export function areAllMeasuresOnSingleAxis(
     secondaryYAxis: IAxisConfig,
 ): boolean {
     const measureCount = getMeasuresFromMdObject(mdObject).length;
-    const numberOfMeasureOnSecondaryAxis = secondaryYAxis ? secondaryYAxis.measures.length : 0;
+    const numberOfMeasureOnSecondaryAxis =
+        secondaryYAxis && secondaryYAxis.measures ? secondaryYAxis.measures.length : 0;
```
Comment (naibinh) [inline src/helpers/MdObjectHelper.ts]: how about `const numberOfMeasureOnSecondaryAxis = get(secondaryYAxis, 'measures.length', 0);`?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5451/ : SUCCESS in 13m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/952/ : SUCCESS in 8s

File: src/helpers/MdObjectHelper.ts (hunk 1 above)
Comment (lvkhanh) [reply src/helpers/MdObjectHelper.ts]: updated

**Comment by naibinh** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2824/ : SUCCESS in 15m 48s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5458/ : SUCCESS in 15m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1327/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1819/ : SUCCESS in 17m 26s

**Pull request merged**